src/clj/rems/auth/oidc.clj

```diff
@@ -15,6 +15,7 @@
        "&client_id=" (getx env :oidc-client-id)
        "&redirect_uri=" (getx env :public-url) "oidc-callback"
        "&scope=openid profile email"
+       (getx env :oidc-additional-authorization-parameters)
```

Comment:
So I can leave the value out of my `config.edn` because this will get a `nil` from `config-defaults.edn`?

Reply:
yep!